src/Agent.Sdk/Util/VssUtil.cs

```diff
@@ -27,6 +27,13 @@ public static void InitializeVssClientSettings(ProductInfoHeaderValue additional
 
             VssClientHttpRequestSettings.Default.UserAgent = headerValues;
             VssClientHttpRequestSettings.Default.ClientCertificateManager = clientCert;
+#if OS_LINUX
```

Comment:
what about OSX?

Reply:
Does OSX also use Curl as the HTTP request backend? If so, I'm good with including OSX as well.

Reply:
https://spin.atomicobject.com/2017/09/28/net-core-osx-libcurl-openssl/

OSX does use Curl as the backend. I'll update the condition.